jdbc-v2/src/main/java/com/clickhouse/jdbc/internal/JdbcConfiguration.java

```diff
@@ -240,7 +240,9 @@ public String getDriverProperty(String key, String defaultValue) {
     public Client.Builder applyClientProperties(Client.Builder builder) {
         builder.addEndpoint(connectionUrl)
                 .setOptions(clientProperties);
-
+        if (clientProperties.containsKey("access_token")) {
```

Comment:
I think we may be need to move this code to the client:
1. create ClientConfigProperties,BEARERTOKEN_AUTH ("bearer_token")
2. handle it in Client#setOption

or create com.clickhouse.jdbc.internal.DriverProperties#BEARER_TOKEN("jdbc_bearer_token")
and use here instead of `access_token`;

Current `access_token` may conflict with existing com.clickhouse.client.api.ClientConfigProperties#ACCESS_TOKEN